ArgoProject/src/systems/CameraSystem.cpp

```diff
@@ -22,7 +22,50 @@ void app::sys::CameraSystem::update(app::time::seconds const & dt)
 		if (camera.target.has_value())
 		{
 			auto const & targetLocation = targetView.get(camera.target.value());
-			camera.position = targetLocation.position - (camera.size / 2.0f);
+			math::Vector2f targetPosition = targetLocation.position;
```

Comment:
```suggestion
			math::Vector2f const & targetPosition = targetLocation.position;
```
this way the compiler can optimise away the `targetPosition`